lib/loader.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+process.emitWarning(
+  'The loader module is an experimental API.',
+  'ExperimentalWarning', undefined,
+  'See https://nodejs.org/api/esm.html'
+);
+
+const Loader = require('internal/loader/Loader');
+
+module.exports = new Loader();
```

Comment:
https://github.com/nodejs/node/pull/16874#discussion_r149658415 remains, commenting since default github UI is hiding it